apps/src/storage/dataBrowser/AddKeyRow.jsx

```diff
@@ -72,31 +73,31 @@ class AddKeyRow extends React.Component {
 
   render() {
     return (
-      <tr id="uitest-addKeyValuePairRow" style={dataStyles.row}>
-        <td style={dataStyles.cell}>
+      <tr id="uitest-addKeyValuePairRow" className={dataStyles.row}>
+        <td className={dataStyles.cell}>
           <input
-            style={dataStyles.input}
+            className={dataStyles.input}
             onChange={this.handleKeyChange}
             onKeyUp={this.handleKeyUp}
             placeholder="enter text"
             value={this.state.key || ''}
           />
         </td>
-        <td style={dataStyles.cell}>
+        <td className={dataStyles.cell}>
           <input
-            style={dataStyles.input}
+            className={dataStyles.input}
             onChange={this.handleValueChange}
             onKeyUp={this.handleKeyUp}
             placeholder="enter text"
             value={this.state.value || ''}
           />
         </td>
-        <td style={dataStyles.addButtonCell}>
+        <td className={classNames(dataStyles.cell, dataStyles.cellAddButton)}>
```

Comment:
this is likely subjective, but i think this case is interestingly different from the button/buttonBlue styles below. here, we have "an 'add' button within a cell," which i think should be expressed slightly differently than "a button that is also blue."

```scss
// data-styles.module.scss

.cell {
  &.addButton {
    // .cell.addButton styles
  }
}

.button {
  &Blue {
    // .buttonBlue styles
  }
}
```

which would then be expressed in JS as...

```javascript
// AddKeyRow.jsx

// .cell.addButton - an 'add' button within a cell
classNames(dataStyles.cell, dataStyles.addButton)

// .buttonBlue - a button that is blue
classNames(dataStyles.button, dataStyles.buttonBlue)
```

thoughts? like i said, this is subjective, so feel free to disagree or think through alternative ways we could express this -- i just think its an interesting semantic difference

Reply:
I wondered about this too. In this case, I don't think it's "an 'add' button within a cell." I think it's "a cell meant for an 'add' button".

Reply:
ah okay, that's slightly different, but i think i still like the approach i suggested. i don't feel too passionately about it, so i'll leave it up to you!

Reply:
Also not passionate about this! I was trying to preserve the semantics I observed here: https://github.com/code-dot-org/code-dot-org/blob/fad0898c61afe8f7a07b396fd9739f6220fdfb5b/apps/src/storage/dataBrowser/dataStyles.js#L16-L28

My understanding right now is that this solution may not work if we're nesting:
```
classNames(dataStyles.cell, dataStyles.addButton)
```

I believe this will add the `.cell` class no problem, but then it will add a class for a non-existent `.addButton` at the root level of the module. I haven't found a way yet to get specify this inner class on element:
```
.cell {
  &.addButton {
    // .cell.addButton styles
  }
}
```

Reply:
i did a quick test in DSCO and the nesting should work fine (only difference is DSCO is on webpack 5 already, but i don't think that should change this specific thing).

this SCSS:
```scss
.cell {
  // .cell styles

  &.addButton {
    // .cell.addButton styles
  }
}
```

results in this SCSS module object being imported into JS:
```js
{
  addButton: <module UUID>,
  cell: <module UUID>
}
```

Reply:
Thank you! I got this to work. :) https://github.com/code-dot-org/code-dot-org/pull/47567/commits/2acbe05a620c13ca7863ce5cee0c2affa6f8a79e

While I was here I also noticed an edit button that was slightly gray instead of white and realized I had missed some nuance in the original style objects for buttons:
https://github.com/code-dot-org/code-dot-org/blob/fad0898c61afe8f7a07b396fd9739f6220fdfb5b/apps/src/storage/dataBrowser/dataStyles.js#L49-L90

Specifically, I thought that white, red, blue, edit, and save were all just types of buttons. What I missed is that edit is a type of white button and save is a type of blue button. I had to correct the nesting and how we were expressing it in the jsx for two affected components. Whew!